options/m_option.c

```diff
@@ -2080,10 +2079,9 @@ static int parse_afmt(struct mp_log *log, const m_option_t *opt,
         return M_OPT_MISSING_PARAM;
 
     if (!bstrcmp0(param, "help")) {
-        mp_info(log, "Available formats:");
+        mp_info(log, "Available formats:\n");
```

Comment:
Why do there need to be newlines here?

Reply:
So I can avoid special cases or more complicated logic in the completion script. Most options print one value per line, so if I make these do that too then the parsing logic stays simple. I didn't see any particular reason to keep them on one line.